modules/ui/map_in_map.js

```diff
@@ -325,7 +325,7 @@ export function uiMapInMap(context) {
         redraw();
 
         var keybinding = d3keybinding('map-in-map')
-            .on(key, toggle);
+            .on([key, '⇧'+key], toggle);
```

Comment:
I'd rather not bind <kbd>Shift-/</kbd> here, because on US keyboards that's <kbd>?</kbd>, which we're saving for #1481